app/client/components/cancel/contributions/contributionsCancellationAmountUpdatedSaved.tsx

```diff
@@ -0,0 +1,58 @@
+import * as Sentry from "@sentry/browser";
+import React from "react";
+import { formatDateStr } from "../../../../shared/dates";
+import {
+  isPaidSubscriptionPlan,
+  MembersDataApiItemContext
+} from "../../../../shared/productResponse";
+import { getMainPlan, isProduct } from "../../../../shared/productResponse";
+import { GenericErrorMessage } from "../../identity/GenericErrorMessage";
+import { SavedBodyProps } from "../stages/savedCancellation";
+
+const ContributionsCancellationAmountUpdatedSaved: React.FC<SavedBodyProps> = ({
+  amount
+}: SavedBodyProps) => {
+  return (
+    <MembersDataApiItemContext.Consumer>
+      {productDetail => {
+        if (!isProduct(productDetail)) {
+          Sentry.captureMessage(
+            "MembersDataApiItem is not a productDetail in ContributionsCancellationAmountUpdateSaved"
```

Comment:
These messages may be a little verbose, perhaps we don't need the `in ContributionsCancellationAmountUpdateSaved` part?

Reply:
I personally don't mind the extra context that this gives us